add captions for images.

sherlynchan opened the pull request.

Atcold commented:
Ehm… You have removed all the empty lines I've added, and added the extra empty lines I've removed.
Can you please just change the captions?

sherlynchan commented:
Hi alf,
Sorry about this. Please check this again. I newed a PR. (as I messed up locally. )
https://github.com/Atcold/pytorch-Deep-Learning/pull/309 <https://github.com/Atcold/pytorch-Deep-Learning/pull/309>
Hope this works.
Best,
Shaoling

> On May 12, 2020, at 3:03 PM, Alfredo Canziani <notifications@github.com> wrote:
> 
> 
> Ehm… You have removed all the empty lines I've added, and added the extra empty lines I've removed.
> Can you please just change the captions?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/Atcold/pytorch-Deep-Learning/pull/306#issuecomment-627533469>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AC2ZR2ZUG5ASLADC6WAQHJTRRGMRRANCNFSM4M66HRKA>.
>

Atcold commented:
Perfect. Thanks!
